src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder method called for userId: {}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("Order completed for userId: {}", userId);
         return order;
```

Comment:
The log message in line 19 should include more details, such as the order details. This will make the log message more informative. For example: logger.info("Order completed for userId: {}. Order details: {}", userId, order);